src/main.cc

```diff
@@ -85,101 +92,157 @@ Global Options:
 
 For more documentation, visit https://github.com/sourcemeta/jsonschema
 )EOF"};
+// Initialize color usage
+bool use_colors = true;
 
 auto jsonschema_main(const std::string &program, const std::string &command,
-                     const std::span<const std::string> &arguments) -> int {
-  if (command == "fmt") {
+                     const std::span<const std::string> &arguments) -> int
+{
+  if (command == "fmt")
+  {
     return sourcemeta::jsonschema::cli::fmt(arguments);
-  } else if (command == "frame") {
+  }
+  else if (command == "frame")
+  {
     return sourcemeta::jsonschema::cli::frame(arguments);
-  } else if (command == "bundle") {
+  }
+  else if (command == "bundle")
+  {
     return sourcemeta::jsonschema::cli::bundle(arguments);
-  } else if (command == "compile") {
+  }
+  else if (command == "compile")
+  {
     return sourcemeta::jsonschema::cli::compile(arguments);
-  } else if (command == "lint") {
+  }
+  else if (command == "lint")
+  {
     return sourcemeta::jsonschema::cli::lint(arguments);
-  } else if (command == "validate") {
+  }
+  else if (command == "validate")
+  {
     return sourcemeta::jsonschema::cli::validate(arguments);
-  } else if (command == "metaschema") {
+  }
+  else if (command == "metaschema")
+  {
     return sourcemeta::jsonschema::cli::metaschema(arguments);
-  } else if (command == "test") {
+  }
+  else if (command == "test")
+  {
     return sourcemeta::jsonschema::cli::test(arguments);
-  } else if (command == "identify") {
+  }
+  else if (command == "identify")
+  {
     return sourcemeta::jsonschema::cli::identify(arguments);
-  } else if (command == "canonicalize") {
+  }
+  else if (command == "canonicalize")
+  {
     return sourcemeta::jsonschema::cli::canonicalize(arguments);
-  } else if (command == "encode") {
+  }
+  else if (command == "encode")
+  {
     return sourcemeta::jsonschema::cli::encode(arguments);
-  } else if (command == "decode") {
+  }
+  else if (command == "decode")
+  {
     return sourcemeta::jsonschema::cli::decode(arguments);
-  } else {
-    std::cout << "JSON Schema CLI - v"
-              << sourcemeta::jsonschema::cli::PROJECT_VERSION << "\n";
-    std::cout << "Usage: " << std::filesystem::path{program}.filename().string()
-              << " <command> [arguments...]\n";
-    std::cout << USAGE_DETAILS;
+  }
+  else
+  {
+    print_success(std::string("JSON Schema CLI - v") + std::string(sourcemeta::jsonschema::cli::PROJECT_VERSION));
+
+    print_success("Usage: " + std::filesystem::path{program}.filename().string() + " <command> [arguments...]");
+
+    print_success(std::string(USAGE_DETAILS));
+
     return EXIT_SUCCESS;
   }
 }
 
-auto main(int argc, char *argv[]) noexcept -> int {
-  try {
+auto main(int argc, char *argv[]) noexcept -> int
```

Comment:
Most of these seem to be unrelated formatting changes?

Reply:
I am going to revert al these formatting. Thankyou.